src/main/webapp/i18n/en/global.json

```diff
@@ -194,7 +195,8 @@
             "cleanupBuildPlan": "Cleanup Build Plan",
             "registerForCourse": "Register For Course",
             "exercise": "Exercise",
-            "scores": "Scores"
+            "scores": "Scores",
+            "manageHints": "Manage hints"
```

Comment:
In my opinion, it is better to add capitalization here `Manage Hints`

Reply:
You are right!